composer.json

```diff
@@ -16,7 +16,7 @@
     "license" : "MIT",
     "require" : {
         "masterminds/html5" : "^2.1.2",
-        "twig/twig" : "^1.10||^2.0",
+        "twig/twig" : "^1.28||^2.0",
```

Comment:
If you want to make Twital really compatible with Twig 2.0, you should upgrade to 1.28. Otherwise we have to test with `method_exists` function.